alerts/charts/values.yaml

```diff
@@ -45,6 +45,16 @@ alerts:
       prometheusOperator: false
       windows: false
 
+  ## Create curated dashboards for generic monitoring (e.g. monitoring Alertmanager)
+  ##
+  dashboards:
+    create: true
+
+    ## Label selectors for the Plutono dashboards to be picked up by Plutono.
+    plutonoSelectors:
+      - name: plutono-dashboard
+        value: '"true"'
```

Comment:
do we need double quoting? If so, why?

Reply:
I made it a `string` because label values do not allow boolean values. And Helm will remove the quotes during template. 

`The edited file had a syntax error: unable to decode "edited-file": json: cannot unmarshal bool into Go struct field ObjectMeta.metadata.labels of type string`

https://github.com/cloudoperators/greenhouse-extensions/blob/9f9c7fe3e6ea084bdbf28c0ae62da0dd3fc81bdd/plutono/charts/values.yaml#L935

https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/